WTWSMS/localisation/WtWSMS_warfare.csv

```diff
@@ -27,7 +27,8 @@ RETTYPE_CUL_ILLY;Balkan Falxmen;Guerriers balkaniques 
 RETTYPE_CUL_RBT;Brito Bellatorum;Brito Bellatorum;;;;;;;;;;;;x
 RETTYPE_CUL_GMR;Germanorum Peditum;Germanorum Peditum;;;Germanorum Peditum;;;;;;;;;x
 #RETTYPE_CUL_PAR;Teutonicus Foederatus;Teutonicus Foederatus;;;Federados Teut�nicos;;;;;;;;;x
-RETTYPE_CUL_PAR;Cataphract;;;;;;;;;;;x
+RETTYPE_CUL_GREPER;Cataphract;;;;;;;;;;;x
```

Comment:
As I explained earlier, I disagree on the name Cataphract and I think it is misleading on the implication that it would be "Greek-Persian", because most cultures aren't Greek and the separation makes little sense.

Reply:
OK, made non-byzantine and non-persian use Clibanarii instead of Cataphract

Reply:
Okay, but still, why are there two different Cataphract retinues?

Reply:
What do you mean?

Reply:
RETTYPE_CUL_BYZ? They have different unit setup

Reply:
It isn't logical to have two separate units with the same name, to me it seems confusing.

Reply:
But the player will never see both of them at once AFAIK

Reply:
No, but there is still a confusion given that all retinues have unique names. I don't see the issue in giving a custom name here though "Oriental Cataphracts", maybe?

Reply:
Good idea

Reply:
Done, RETTYPE_CUL_GREPER is now Oriental Cataphract